src/main/java/com/greenfoxacademy/goddesstribesbackend/controllers/BuildingController.java

```diff
@@ -93,21 +93,20 @@ public BuildingController(KingdomService kingdomService, ProductionService produ
       return ResponseEntity.status(404).body(new ErrorMessage("No building with such id found in your kingdom!"));
     }
 
-
-    if (!buildingService.isValidLevel(levelDTO.getLevel(), buildingToUpgrade.getLevel(), kingdom.getId(), buildingToUpgrade.getType())){
+    if (!buildingService.isValidLevel(levelDTO.getLevel(), buildingToUpgrade.getLevel(), kingdom.getId(), buildingToUpgrade.getType())) {
```

Comment:
It might worth breaking these very long line into other lines to make the file prettier.

Reply:
Bocsánat, hadd írjam magyarul: Ezt nem én írtam, hanem Judit...Barni jóváhagyta...nem szeretnék beleírni más kódjába...